src/AccessibilityInsights.SharedUx/Controls/SettingsTabs/ApplicationSettingsControl.xaml

```diff
@@ -236,7 +236,7 @@
                     <ColumnDefinition Width="151"/>
                     <ColumnDefinition Width="180"/>
                 </Grid.ColumnDefinitions>
-                <Label Name="lblFontSizeLabel" Grid.Column="0" Content="Font Size" Padding="0" Style="{StaticResource LblText}" VerticalAlignment="Center"/>
+                <Label Name="lblFontSizeLabel" Grid.Column="0" Content="{x:Static Properties:Resources.ApplicationSettingsControl_fontsize}" Padding="0" Style="{StaticResource LblText}" VerticalAlignment="Center"/>
```

Comment:
We typically use PascalCasing, so this would be `ApplicationSettingsControl_FontSize`